include/boost/math/special_functions/nonfinite_num_facets.hpp

```diff
@@ -223,7 +223,6 @@ namespace boost {
           *it = fill;
       }
 
-    private:
```

Comment:
I'm confused; why remove the `private` keyword here?

Reply:
Ditto as below

Reply:
It is still private. Jus that the private keyword is already above